dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201014UpdateColumnsValuesInIdentifierTable.java

```diff
@@ -11,28 +12,58 @@
 public class Task201014UpdateColumnsValuesInIdentifierTable implements StartupTask {
 
     protected static final String UPDATE_CONTENTLETS =
-            "update identifier set owner=q.i_owner, create_date=q.create_date, asset_subtype=q.asset_subtype from"
-            + "(select cont.identifier, cont.mod_user i_owner, inode.idate create_date, "
-            + "struc.velocity_var_name asset_subtype from contentlet cont, "
-            + "inode inode, structure struc "
-            + "where inode.inode=cont.inode "
-            + "and cont.structure_inode = struc.inode) q where id=q.identifier";
+            "update identifier as ident, "
+                    + "(select cont.identifier, cont.mod_user i_owner, inode.idate create_date, struc.velocity_var_name asset_subtype "
+                    + "from contentlet cont, inode inode, structure struc where inode.inode=cont.inode and cont.structure_inode = struc.inode) "
+                    + "as q set ident.owner=q.i_owner, ident.create_date=q.create_date, ident.asset_subtype=q.asset_subtype  where ident.id=q.identifier";
 
-    private static final String UPDATE_FOLDER = "update identifier set owner=q.i_owner, create_date=q.create_date from "
-            + "(select identifier, owner i_owner, idate create_date from folder, inode "
-            + "where inode.inode=folder.inode) q where id=q.identifier";
+    protected static final String UPDATE_CONTENTLETS_POSTGRESQL =
+            "update identifier set owner=q.i_owner, create_date=q.create_date, asset_subtype=q.asset_subtype from "
+                    + "(select cont.identifier, cont.mod_user i_owner, inode.idate create_date, "
+                    + "struc.velocity_var_name asset_subtype from contentlet cont, "
+                    + "inode inode, structure struc "
+                    + "where inode.inode=cont.inode "
+                    + "and cont.structure_inode = struc.inode) q where id=q.identifier";
 
-    private static final String UPDATE_CONTAINERS="update identifier set owner=q.i_owner, create_date=q.create_date from "
-            + "(select identifier, owner i_owner, idate create_date from dot_containers, inode "
-            + "where inode.inode=dot_containers.inode) q where id=q.identifier";
+    private static final String UPDATE_FOLDER = "update identifier as ident, "
+            + "(select identifier, owner i_owner, idate create_date from folder, inode where inode.inode=folder.inode) as q "
+            + "set ident.owner=q.i_owner, ident.create_date=q.create_date "
+            + "where ident.id=q.identifier";
```

Comment:
Codacy found an issue: [The String literal "where ident.id=q.identifier" appears 4 times in this file; the first occurrence is on line 31](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6379663)